src/all_types.hpp

```diff
@@ -3229,6 +3231,14 @@ struct IrInstructionTypeInfo {
     IrInstruction *type_value;
 };
 
+struct IrInstructionHasField {
+    IrInstruction base;
+
+    IrInstruction *container_type;
+    Buf *field_name_buffer;
```

Comment:
I don't think the existence of this field is correct. I think you can follow this pattern: 269a53b6afb052ec09f7b26db68219cfeaa0460e

Reply:
I was following another existing pattern.